extensions/spring-cloud-config-client/runtime/src/test/java/io/quarkus/spring/cloud/config/client/runtime/SpringCloudConfigClientGatewayTest.java

```diff
@@ -78,7 +78,7 @@ private static SpringCloudConfigClientConfig configForTesting() {
         springCloudConfigClientConfig.url = "http://localhost:" + MOCK_SERVER_PORT;
         springCloudConfigClientConfig.label = Optional.of("master");
         springCloudConfigClientConfig.connectionTimeout = Duration.ZERO;
-        springCloudConfigClientConfig.readTimeout = Duration.ZERO;
+        springCloudConfigClientConfig.readTimeout = Duration.ofSeconds(10);
```

Comment:
Why did you change this?

Reply:
ah, my fault, it was during the tests, I will undo it!

Reply:
It doesn't seem like you undid this one :)

Reply:
today is not my day. It should be ok now. Thanks!! :-)